pyproject.toml

```diff
@@ -0,0 +1,77 @@
+[tool.mypy]
+follow_imports = "normal"
+warn_redundant_casts = true
+warn_unused_ignores = true
+disallow_any_generics = true
+check_untyped_defs = true
+no_implicit_reexport = true
+pretty = true
+
+# to be switched to true
+disallow_incomplete_defs = false
+disallow_untyped_defs = false
+disallow_untyped_calls = false
+
+[tool.isort]
+profile = "black"
+skip_gitignore = true
+line_length = 100
+
+[tool.black]
+line-length = 100
+
+[tool.coverage.run]
+branch = false
+omit = ["tests/*"]
+
```

Comment:
Ik zou hier ook `data_file = "/tmp/.coverage"` neerzetten. Dan kan `export COVERAGE_FILE="/tmp/.coverage"` weg uit `test.sh`.

Reply:
`data_file` config onder coverage.report bestaat helaas niet in pyproject